src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -425,15 +431,15 @@ object SbtWeb extends AutoPlugin {
     withWebJarExtractor(target, cache, classLoader) {
       (e, to) =>
         e.extractAllNodeModulesTo(to)
-    }.***.get
+    }.**(AllPassFilter).get
   }
 
   private def generateWebJars(target: File, lib: String, cache: File, classLoader: ClassLoader): Seq[File] = {
     withWebJarExtractor(target / lib, cache, classLoader) {
       (e, to) =>
         e.extractAllWebJarsTo(to)
     }
-    target.***.get
+    target.**(AllPassFilter).get
```

Comment:
`allPaths` doesn't exist in sbt 0.13.16 (as far as I can see), this is needed to cross build.